lms/static/js/commerce/track_ecommerce_events.js

```diff
@@ -1,7 +1,8 @@
 /**
  *
  * A library of helper functions to track ecommerce related events.
- *
+ * See here for the full list of upsell links and how to view these events in snowflake:
```

Comment:
:8ball: I might leave out the `how to view these events in snowflake` bit since that might be different for other edx instances/we might move to a different data source/you can also view it in Segment/etc.